docs/dev-env.md

```diff
@@ -175,8 +175,8 @@ $ docker run -it -v $PWD/demo:/var/fedimint -p 17240:17240 fedimintd:iqviraxy2cz
 write the files to the host file-system (e.g. config generation). `-p` is used to bind the host's port 17240 as the
 container's port 17240.
 
-To generate federation configs and run the federation please see the `run_dkg` and `start_federation` helper functions in [lib.sh](https://github.com/fedimint/fedimint/blob/master/scripts/lib.sh), as well as the `--help` output of
-`distributedgen` and `fedimintd`. If you'd like to run function like `run_dkg` and `start_federation` from inside nix shell, run `source scripts/build.sh` and then `source scripts/lib.sh` and they will be available.
+To generate federation configs and run the federation please see the `run_dkg` and `start_federation` as well as the `--help` output of
```

Comment:
`run_dkg` and `start_federation` are also gone. 
soon there will be `devimint run-dkg` and `devimint start-federation`